### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by tijanalego**
I've made a new change in the logging function. Please review.

### CODE REVIEW

**Comment by tijanalego** [comment]
hello

**Comment by NielsMKristensen** [comment]
OK i will check it out

File: NielsPullStuff.md [hunk 1]
```diff
@@ -0,0 +1 @@
+#HELOOOOOOOOO
```
Comment (tijanalego) [inline NielsPullStuff.md:1]: This API call is not using proper auth

Comment (NielsMKristensen) [reply NielsPullStuff.md:1]: but it is

Comment (NielsMKristensen) [reply NielsPullStuff.md:1]: TESTING Comments

**Review by NielsMKristensen — APPROVED** [review]
looks good awsome committer

**Pull request merged**